CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## [0.0.33] - 2023-08-07
+
+### Added
+
+- [#218](https://github.com/Azure/draft/pull/218) Add gradle wrapper detection
+- [#217](https://github.com/Azure/draft/pull/217) Add python entrypoint detection
```

Comment:
📓 💡 Nice and thanks! Please Note - there are couple of PRs not mentioned in the changelog, like `219` mentioning this to make sure we baseline the reason so we can make that a norm in case changelog don't contain end to end artefact information which is getting released. or `220` or `225` #218 #220 , #225

Reply:
good point i've added these to the changelog